### PR TITLE
Add script to generate Cross CPU pipeline

### PR DESCRIPTION
**Pull request by pb8o**
## Changes

...

## Reason

...

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

~- [ ] If a specific issue led to this PR, this PR closes the issue.~
~- [ ] The description of changes is clear and encompassing.~
~- [ ] Any required documentation changes (code and docs) are included in this PR.
~- [ ] API changes follow the [Runbook for Firecracker API changes][2].
~- [ ] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.
~- [ ] New `TODO`s link to an issue.~
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

~- [ ] This functionality can be added in [`rust-vmm`][1].~

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Review by kalyazin — DISMISSED** [review]

File: .buildkite/pipeline_cross.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            timeout=30,
+            artifact_paths="snapshots/**/*",
+            instances=snap_instances,
+            platforms=DEFAULT_PLATFORMS,
+        )
+    )
+    groups.append("wait")
+
+    kvs = ["linux_4.14", "linux_5.10"]
+    instances_x86_64 = ["m5d.metal", "m6i.metal", "m6a.metal"]
```
Comment (kalyazin) [inline .buildkite/pipeline_cross.py:54]: Is this going to test AMD <-> Intel snapshots too? I don't think it is supported?

Comment (pb8o) [reply .buildkite/pipeline_cross.py:54]: Yeah, m6i -> m6a doesn't work... though m6a -> m6i seems to work. Should we skip the tests completely or assert that they fail/work when appropiate?

Comment (kalyazin) [reply .buildkite/pipeline_cross.py:54]: I don't think it makes much sense to assert they fail. I guess we can have an exception list of what we know is not working (either due to limitations or by design).

**Review by kalyazin — DISMISSED** [review]

File: .buildkite/pipeline_cross.py (hunk 1 above)
Comment (pb8o) [reply .buildkite/pipeline_cross.py:67]: done

**Review by kalyazin — APPROVED** [review]

**Review by sudanl0 — APPROVED** [review]

**Pull request merged**